staging/src/k8s.io/client-go/tools/cache/delta_fifo_test.go

```diff
@@ -98,14 +98,15 @@ func TestDeltaFIFO_replaceWithDeleteDeltaIn(t *testing.T) {
 	f := NewDeltaFIFO(testFifoObjectKeyFunc, keyLookupFunc(func() []testFifoObject {
 		return []testFifoObject{oldObj}
 	}))
+	f.clientUnderstandsReplaced = true
```

Comment:
I would actually like to leave this test entirely as-is, and make a copy that sets this option and verifies the Replaced delta type

Reply:
same comment applies to the other existing unit tests that were modified